charts/graphite/templates/deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "graphite.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "graphite.name" . }}
+    helm.sh/chart: {{ include "graphite.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  strategy:
+    type: Recreate
```

Comment:
This might be configurable.

Reply:
will be changed to statefullset anyway...